### ISSUE TITLE
Unable to find Language in where clause

### ISSUE THREAD

**Issue by salmanamemon** [valu-digital/wp-graphql-polylang#63]
Hi all,
I have installed all the plugin, 
Acf Pro,
Polylang 3.1.2
graphql for polylang
acf for graphql

language is not there when looking into the where clause am i doing something wrong?

**Comment by esamattis** [comment]
What's the query you are trying to do?

Some things to check:

- Have you configured the Polylang languages?
- Have you marked the post type / taxonomy to be translateable in the Polylang settings?

**Comment by salmanamemon** [comment]
Yes all setting are done. When using with ACF Pro there is no WHERE clause for Language.

**Comment by simplenotezy** [comment]
Guess it's related to: https://github.com/valu-digital/wp-graphql-polylang/issues/66
